test/quantization/test_quantize_fx.py

```diff
@@ -1120,46 +1120,86 @@ def forward(self, x):
                     quantized_nodes[dim])
 
     def _test_activation_impl(
-            self, float_module, float_op, quantized_module, quantized_op):
-        ''' Test for activation op(with inplace options), float_op can be
+            self, float_module, float_op,
+            quantized_module, quantized_op,
+            extra_module_kwargs=None, extra_op_kwargs=None):
+        r"""Test for activation op(with inplace options), float_op can be
         torch op or functional op
-        '''
+        """
+        if extra_module_kwargs is None:
+            extra_module_kwargs = {}
+        if extra_op_kwargs is None:
+            extra_op_kwargs = {}
+
         class M(torch.nn.Module):
-            def __init__(self, is_module, inplace):
-                super(M, self).__init__()
+            def __init__(self, is_module):
+                super().__init__()
                 self.is_module = is_module
-                self.inplace = inplace
                 if self.is_module:
-                    self.op = float_module(self.inplace)
+                    self.op = float_module(**extra_module_kwargs)
                 else:
                     self.op = float_op
 
             def forward(self, input):
                 if self.is_module:
                     return self.op(input)
                 else:
-                    return self.op(input, self.inplace)
+                    return self.op(input, **extra_op_kwargs)
 
-        options = itertools.product([True, False], [True, False], self.static_quant_types)
+        options = itertools.product([True, False], self.static_quant_types)
         quantized_nodes = {
             # is_module
             True: ns.call_module(quantized_module),
             False: ns.call_function(quantized_op),
         }
 
-        for is_module, is_inplace, quant_type in options:
+        for is_module, quant_type in options:
             self.checkGraphModeFxOp(
-                M(is_module, is_inplace), self.img_data_2d,
+                M(is_module), self.img_data_2d,
                 quant_type, quantized_nodes[is_module])
 
     def test_hardswish(self):
-        self._test_activation_impl(nn.Hardswish, F.hardswish, nnq.Hardswish, torch.ops.quantized.hardswish)
+        for inplace in [True, False]:
+            extra_kwargs = {'inplace': inplace}
+            self._test_activation_impl(
```

Comment:
Can we pass `inplace` as an argument to `_test_activation_impl` to avoid this extra look in the test?

Reply:
not sure I understand, I think current way of passing argument is more general than just passing an inplace argument

Reply:
Oh, my suggestion was to move the loop over `inplace` inside `_test_activation_impl`.

Reply:
oh, we can't do that since not all activation functions has `inplace` argument